pkg/api/v1/types.go

```diff
@@ -3024,6 +3024,8 @@ const (
 	ResourceServices ResourceName = "services"
 	// ReplicationControllers, number
 	ResourceReplicationControllers ResourceName = "replicationcontrollers"
+	// ReplicaSets, number
+	ResourceReplicaSets ResourceName = "replicasets"
```

Comment:
yeah, i was still debating the same.  I have no objection to changing it to be `groupName/resourceName`